src/Tools/Source/RunTests/Program.cs

```diff
@@ -348,67 +331,7 @@ private static ITestExecutor CreateTestExecutor(Options options)
                 useHtml: options.UseHtml,
                 test64: options.Test64,
                 testVsi: options.TestVsi);
-            var processTestExecutor = new ProcessTestExecutor(testExecutionOptions);
-            if (!options.UseCachedResults)
-            {
-                return processTestExecutor;
-            }
-
-            // The web caching layer is still being worked on.  For now want to limit it to Roslyn developers
-            // and Jenkins runs by default until we work on this a bit more.  Anyone reading this who wants
-            // to try it out should feel free to opt into this.
-            IDataStorage dataStorage = new LocalDataStorage();
-            if (CanUseWebStorage())
-            {
-                dataStorage = new WebDataStorage();
-            }
-
-            return new CachingTestExecutor(processTestExecutor, dataStorage);
-        }
-
-        /// <summary>
-        /// Order the assembly list so that the largest assemblies come first.  This
-        /// is not ideal as the largest assembly does not necessarily take the most time.
-        /// </summary>
-        /// <param name="list"></param>
-        private static IOrderedEnumerable<string> OrderAssemblyList(IEnumerable<string> list)
```

Comment:
was this just dead code?

Reply:
Yeah I believe an earlier PR from @jmarolf made this dead and we didn't notice